osu.Game/Skinning/LegacyManiaSkinConfiguration.cs

```diff
@@ -54,7 +54,7 @@ public LegacyManiaSkinConfiguration(int keys)
         public float MinimumColumnWidth
         {
             get => minimumColumnWidth ?? ColumnWidth.Min();
-            set => minimumColumnWidth = value;
```

Comment:
sure we don't want to do this in the actual parsing logic (`LegacyManiaSkinDecoder`)? seems weird to be doing it here.

Reply:
Yeah I don't mind